### DISCUSSION TITLE
Downloading magic wand API in c vs

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#7421] [Help]
### ImageMagick version

7

### Operating system, version and so on

windows 10 on vs 2022

### Description

I am trying to write a c program that uses magic wand to turn list of images into a gif but the `#include <MagickWand/MagickWand.h>` does not work for me because it says it cant find MagicWand and I tried to find the download in the website but couldnt find it

**Comment by snibgo** [comment]
If you want to download individual source files, click on `code` at top-left of this page, then on `MagickWand`, and so on.

But I suppose you need more than one file, and you need to link your program with the ImageMagick library. To download the source files and build with Visual Studio, see https://imagemagick.org/script/install-source.php#windows .

However, I build on Windows with Cygwin tools. See:
[Compiling IM with Cygwin](http://im.snibgo.com/compim.htm)
[Compiling the MagickWand demonstration programs](http://im.snibgo.com/compdemo.htm)
